### ISSUE TITLE
Chapter 1 Problem 7 missing cases

### ISSUE THREAD

**Issue by charmedrabbit** [fifthist/Introduction-To-Probability-Blitzstein-Solutions#25]
I believe there are a couple missing cases for problem 7 in chapter 1 (counting chess wins), part c (src/chapters/1/sections/counting/problems/7.tex). In particular I think you miss the cases for "A" winning 2 games and drawing 4 (with the last game being a win), and the case for "A" winning the last game and drawing the first 6.
Combinatorics is very difficult for me to get correct, so I could be wrong about this. That said, my reasoning is the following:
In order to last 7 games, "A" must achieve their last point (a win or a draw) on game 7.
In order to get 4 points, "A" must win 4 games, or win 3 and draw 2, or win 2 and draw 4, or win 1 and draw 6.
For the case of winning 4, the last must be a win, leaving ${6 \choose 3}$ for the 3 wins in the first 6 games.
For the case of winning 3, the last can be a win, leaving ${6 \choose 2}{4 \choose 2}$ for the remaining wins and draws. Alternately, the last can be a draw, leaving ${6 \choose 3}{3 \choose 1}$ for the remaining wins and draws.
For the case of winning 2, the last can be a win (I think you missed this), leaving ${6 \choose 1}{5 \choose 4}$. Alternately the last can be a draw, leaving ${6 \choose 2}{4 \choose 3}$.
Finally, for the case of winning 1, the last can be a win (I think you missed this), leaving ${6 \choose 6}$ for the draws. Alternately, the last can be a draw, leaving ${6 \choose 1}{5 \choose 5}$ for the remaining win and draws.

**Comment by willhunt** [comment]
Yes, the answer to this question should be 267 (source[ edx course](https://learning.edx.org/course/course-v1:HarvardX+STAT110x+2T2022/home)) but comes to 236 in this soultion.
